src/components/SpruceForm/LeafyGreenWidgets.tsx

```diff
@@ -299,6 +303,51 @@ export const LeafyGreenTextArea: React.FC<WidgetProps> = ({
   );
 };
 
+export const LeafyGreenSegmentedControl: React.FC<WidgetProps> = ({
+  id,
+  label,
+  onChange,
+  options,
+  value,
+}) => {
+  const {
+    "aria-controls": ariaControls,
+    "data-cy": dataCy,
+    enumOptions,
+  } = options;
+
+  if (!Array.isArray(enumOptions)) {
+    console.error("Non-Array passed into Segmented Control");
+    return null;
+  }
+
+  const idPrefix = id.substring(0, id.lastIndexOf("_"));
```

Comment:
i'm curious about this transformation?  Why is this necessary?

Reply:
Good point, I was experimenting with programmatically computing the IDs that the segmented control controls, but it's way more straightforward to specify them manually. Removed!